package.json

```diff
@@ -2,7 +2,6 @@
   "name": "aries",
   "private": true,
   "workspaces": [
-    "aries-core",
-    "aries-site"
+    "packages/*"
```

Comment:
I'm not convinced about the 'packages' directory, I think at this point it adds unnecessary abstraction we don't need. 
It also might affect unpleasantly the way `hpe-server` connects with aries-core.

Reply:
I don't think it would have a negative effect, but I'll need to confirm that. Regardless, this packages/* structure mirrors what would be done if we used Lerna and seems to be a common pattern I've seen across other projects using yarn workspaces. I do agree that maybe it's an unnecessary abstraction considering we only have two workspaces at the moment, but I don't think it will have negative affects. I will look for some more resources.